ansible/group_vars/all

```diff
@@ -102,6 +102,10 @@ dashd_minimumdifficultyblocks: 4032
 # You can provide your own seed node in network config. May be useful if you want to connect to another network
 dashd_seednode: '{{ hostvars[groups.seed_nodes[0]]["public_ip"] if groups.seed_nodes else none }}'
 
+#Do the same for walletnode so we can identify later
+dashd_walletnode: '{{ hostvars[groups.seed_nodes[0]]["public_ip"] if groups.wallet_nodes else none }}'
```

Comment:
Shouldn't this be `hostvars[groups.wallet_nodes[0]]["public_ip"]`? I'm not clear what this line does, can you explicitly state it in the comment, I'm not clear what "the same" is, and when "later"is.